.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Changes
+
+<!-- Summary of your changes that are easy to understand -->
+
+## Tests
+
+<!-- How is this tested? -->
+
+## Jira Ticket
+
+<!--
+For linking Jira ticket with PR. This will show up on the ticket.
+Not to be added in PR title as these will need to be removed in the change logs.
+-->
```

Comment:
Would be great if you can put in examples here. We do not want people to commit full links to our internal JIRA.

Also, for open source repos, we do not track all issues in JIRA. Infact we want to minimise JIRA exposure and track as much of the work in github issues as possible (so that users have better visibility). Could you add a section for this with examples as well? Wdyt about making it higher priority than JIRA in the template?

Reply:
Thanks for the review. Yeah agree, we shouldn't have people add full links, just adding the number should be good since jira automatically picks it up, example just writing:

## Ticket Number (if any)
DECO-613

should be enough. I did the same in this PR and it links it:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/88379306/226908443-9a0c82df-b48c-4563-933a-d257f6109eb3.png">

> github issues as possible (so that users have better visibility). Could you add a section for this with examples as well? Wdyt about making it higher priority than JIRA in the template?
I think having another section for open source issues should be okay but I was thinking on having a single section where people can link (if open source) / mention (if internal jira) the tickets to make PR template less bulky.

Reply:
updated the PR